src/app/qrcode/qr-url/page.tsx

```diff
@@ -0,0 +1,289 @@
+  const [organizationOptions, setOrganizationOptions] = useState<
+    null | Organization[]
+  >(null);
+  const [organizationValue, setOrganizationValue] =
+    useState<null | Organization>(null);
+  const [domainValue, setDomainValue] = useState('');
+  const [domainOptions, setDomainOptions] = useState<null | string[]>(null);
+  const { meProfile, isAuthStatusReady } = useContext(AuthContext);
+
+  useEffect(() => {
+    if (!meProfile) return;
+
+    (async () => {
+      try {
+        const organizationOptionsInitial = await meService.getOrganization();
+        setOrganizationOptions(organizationOptionsInitial);
+        setDomainOptions(organizationOptionsInitial[0].domains);
+        setOrganizationValue(organizationOptionsInitial[0]);
+        setDomainValue(organizationOptionsInitial[0].domains[0]);
+      } catch (e: any) {
+        console.log(e.message);
+      }
+    })();
+  }, [meProfile]);
+
+  useEffect(() => {
+    if (organizationValue) {
+      setDomainOptions(organizationValue.domains);
+      setDomainValue(organizationValue.domains[0]);
+    }
+  }, [organizationValue]);
+
+  const handleChange = (mode: string) => {
+    switch (mode) {
+      case 'organization':
+        return (value: Organization) => {
+          setOrganizationValue(value);
+        };
+      case 'domain':
+        return (value: string) => {
+          setDomainValue(value);
+        };
+      default:
+        return () => {};
+    }
+  };
+  const consoleLog = () => {
+    console.log(meProfile);
+    console.log(organizationOptions);
+    console.log(organizationValue);
+    console.log(domainOptions);
+  };
+  const eventCategory = {
+    _id: '1',
+    name: 'Event',
+    color: '#ff0000',
+    organization: 'org1',
+    urls: ['url1', 'url2'],
+  };
+  const favCategory = {
+    _id: '2',
+    name: 'Favourite',
+    color: '#ff0000',
+    organization: 'org2',
+    urls: ['url1', 'url2'],
+  };
+  return (
+    <>
+      <div className='relative flex flex-col items-center overflow-hidden leading-[1.2] text-primary'>
+        <div className='md:max-w-[1000px] lg:flex lg:flex-col lg:items-center'>
+          <div className='text-center'>
+            <h1 className='mt-[80px] text-[36px] font-[700] leading-[65px] md:mt-[168px] md:text-[48px] lg:text-[60px]'>
+              <span className=' md:inline'>Fessior</span> QR Generator
+            </h1>
+            <p className='hidden leading-[24px] md:block md:text-[20px] lg:mb-0 lg:mt-3 lg:text-[24px]'>
+              Convenience, efficiency, and versatility:{' '}
+              <br className='md:hidden' /> QR Code Management Made Easy
+            </p>
+          </div>
+          {/* {isAuthStatusReady ? ( */}
+          {/* <> */}
+          <div className='mx-auto my-5 flex w-[100%] max-w-[360px] text-[16px] font-[500] md:my-6 md:max-w-[416px] md:text-[20px]'>
+            <Button
+              image='/icons/link-qr-choosen.svg'
```

Comment:
```suggestion
              image='/icons/link-qr-chosen.svg'
```
Grammar of `choosen`